apps/web/src/stores/documentLogWithMetadata.ts

```diff
@@ -3,13 +3,19 @@ import { ROUTES } from '$/services/routes'
 import useSWR, { SWRConfiguration } from 'swr'
 
 export default function useDocumentLogWithMetadata(
-  documentLogUuid?: string | null,
+  {
+    documentLogUuid,
+    onFetched,
+  }: {
+    documentLogUuid?: string | null
+    onFetched?: (data: DocumentLogWithMetadata) => void
+  },
   opts?: SWRConfiguration,
 ) {
   return useSWR<DocumentLogWithMetadata>(
     ['documentLogWithMetadata', documentLogUuid],
     async () => {
-      if (!documentLogUuid) return []
+      if (!documentLogUuid) return undefined
```

Comment:
This was returning one item but when empty was returning an empty array. I think is wrong

Reply:
💯